lib/project_types/script/layers/application/create_script.rb

```diff
@@ -27,10 +28,15 @@ def create_project(ctx, language, script_name, extension_point)
               extension_point_type: extension_point.type,
               script_name: script_name
             )
+            ScriptProject.current
+          end
+
+          def install_dependencies(ctx, language, script_name, extension_point, project)
+            task_runner = Infrastructure::TaskRunner.for(ctx, language, script_name, project)
```

Comment:
I see the change in `push_script.rb`.  Did you forget to change it here?

Reply:
Nice catch 🙏